client/src/components/Problem/ProblemShow.js

```diff
@@ -46,6 +48,14 @@ class ProblemShow extends Component {
         }).isRequired
     }
 
+    constructor(props) {
+        super(props);
+        this.state = {
+            accepted: null,
```

Comment:
We never actually use `this.state.accepted`, so we can probably omit this unless it causes bugs